Frends.Community.PowerShell.Tests/PowerShellTests.cs

```diff
@@ -144,7 +147,7 @@ public void RunCommandAndScript_ShouldUseSharedSession()
                 new RunOptions
```

Comment:
https://github.com/CommunityHiQ/Frends.Community.PowerShell/blob/a0b1eb70068bbf2bb166cb1532f44306342d7468/Frends.Community.PowerShell.Tests/PowerShellTests.cs#L141
You're not using the variable `result1`, so you can omit return of the method call.